### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SerDmitriy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,82 +2,27 @@
...
-	<link rel="stylesheet" href="./css/styles.css">
-	<script src="./js.js" defer></script>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="author" content="Dmitriy Davydov">
+    <title>JS test</title>
+    <!-- <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Lato:300,400,700;subset=cyrillic">
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Montserrat:300,400,700&amp;subset=cyrillic">
+    <link rel="stylesheet" href="./css/styles.css"> -->
```
Comment (Abjuk) [inline index.html:12]: remove commented code

File: js.js [hunk 2]
```diff
@@ -1,114 +1,126 @@
...
+  let question = document.createElement('div');
+  question.innerText = `${counterQuestions + 1}. ${mainDb[counterQuestions].question}`;
+  mainForm.appendChild(question);
+}
 
-    /*if (rightCheckbox != undefined && mainDb[counterQuestions].rightAnswer == rightCheckbox[0].nextElementSibling.innerText) {
-        counterCorrectQuestions++
-    }*/   // there should work rule of short hand operations in "if" comparemant
+function showAnswers() {                            //-----------------------//-------------------------//-------------------------//-------------------------
+  mainDb[counterQuestions].answers.forEach((item, index, array) => {
```
Comment (Abjuk) [inline js.js:94]: remove commented code

File: index.html (hunk 1 above)
Comment (SerDmitriy) [reply index.html:12]: Done

File: js.js (hunk 2 above)
Comment (SerDmitriy) [reply js.js:94]: done

**Review by SerDmitriy — COMMENTED** [review]
fixed

**Pull request merged**